.evergreen/download-mongodb.sh

```diff
@@ -504,3 +505,31 @@ download_and_extract ()
    ./mongodb/bin/mongod --version
    cd -
 }
+
+# download_and_extract downloads a requested MongoDB server package.
+# If the legacy shell is not included in the download, the legacy shell is also downloaded from the 6.0 package.
+download_and_extract ()
+{
+   MONGODB_DOWNLOAD_URL=$1
+   EXTRACT=$2
+
+   download_and_extract_package "$MONGODB_DOWNLOAD_URL" "$EXTRACT"
+
+   if [ ! -e $DRIVERS_TOOLS/mongodb/bin/mongo ]; then
+      # The legacy mongo shell is not included in server downloads of 6.0.0-rc6 or later. Refer: SERVER-64352.
+      # Some test scripts use the mongo shell for setup.
+      # Download 6.0 package to get the legacy mongo shell as a workaround until DRIVERS-2328 is addressed.
```

Comment:
If 6.0 does not include the legacy shell don't we need to download it from 5.0?

Reply:
Oof, good point. Updated to use 5.0.